tests/test_mesh.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import trimesh
+import numpy as np
+import os
+
+import genesis as gs
+import genesis.utils.mesh as mu
+import genesis.utils.gltf as gltf_utils
+
+
+@pytest.mark.parametrize("glb_file", ["assets/cube.glb", "assets/combined_srt.glb", "assets/combined_transform.glb"])
```

Comment:
For now we just move all the assets in `genesis/assets`. It is not ideal I know. I will change this latter.